modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,97 @@
 
-[id="stable-4-channel_{context}"]
-== stable-4 channel
-Releases are added to the `stable-4` channel after passing all tests.
+[NOTE]
+Because the time required to obtain a statistically significant number of data points varies based on many factors we offer no SLA for the delay duration between fast and stable channels. Please see link:strategies_{context}[Fast and stable channel use and strategies] for more information.
 
-You can use the `stable-4` channel to upgrade from a previous minor version of {product-title}.
-endif::openshift-origin[]
 
-ifndef::openshift-origin[]
+Newly installed clusters default to utilizing stable channels.
 
 [id="eus-4y-channel_{context}"]
 == eus-4.y channel
 
-In addition to the stable channel, all even-numbered minor versions of {product-title} offer an link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). These EUS versions extend the Full and Maintenance support phases for customers with Standard and Premium Subscriptions to 18 months.
-
-Although there is no difference between `stable-4.y` and `eus-4.y` channels until {product-title} 4.y transitions to the EUS phase, you can switch to the `eus-4.y` channel as soon as it becomes available.
-
-When upgrades to the next EUS channel are offered, you can switch to the next EUS channel and upgrade until you have reached the next EUS version.
-
-This upgrade process does not apply for the `eus-4.6` channel.
+In addition to the stable channel, all even-numbered minor versions of {product-title} offer link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). Releases which are promoted to the stable channel are also simultaneously promoted to the EUS channels. The primary purpose of the EUS channels is to serve as a convenience for clusters performing an EUS-to-EUS update.
 
 [NOTE]
 ====
 Both standard and non-EUS subscribers can access all EUS repositories and necessary RPMs (`rhel-*-eus-rpms`) to be able to support critical purposes such as debugging and building drivers.
 ====
 
+== candidate-{product-version} channel
+
+The `candidate-{product-version}` channel contains releases as soon as they're produced, prior to being declared GA. Releases present only in the candidate channels have not completed full Quality Assurance testing and lack Red Hat Support. Therefore releases present only in the candidate channels are only suitable for testing purposes where destroying and recreating a cluster is acceptable. Additionally, we provide no guarantee that these releases will be offered supported update paths to GA versions.
+
+ifdef::openshift-origin[]
+[id="stable-4-channel_{context}"]
+== stable-4 channel
+Releases are added to the `stable-4` channel after passing all tests and stable-4 is the only supported channel.
 endif::openshift-origin[]
 
+
+
 [id="upgrade-version-paths_{context}"]
-== Upgrade version paths
+== Update version paths
```

Comment:
I'd sort of like to get rid of it because it's pretty graph centric and if we're trying to just re-orient toward update recommendations maybe it's no longer necessary.

@wking @LalatenduMohanty thoughts?

Reply:
A lot of the other sections are more release-focused (nodes), and this section is update-focused (edges).  I think there's some utility to have a section focused on positive update recommendations to remind everyone of why we're here before moving on to talk about recommendation-removal and update risks.

I agree that the current approach within the section is graph-y.  Perhaps we could rephrase to be more specifically about a single hypothetical cluster running `{product-version}.0` in our hypothetical channel content?

Reply:
The section title `Update version paths` does not seem right here. We should rename as `Upgrade recommendations in the channel`. The content seems relevant to me.